api/v1alpha1/database_types.go

```diff
@@ -233,9 +233,17 @@ type SLAResponse struct {
 
 // +kubebuilder:object:generate:=false
 type DatabaseNode struct {
-	Id               string `json:"id"`
-	Name             string `json:"name"`
-	DatabaseServerId string `json:"dbServerId"`
+	Id               string         `json:"id"`
+	Name             string         `json:"name"`
+	DatabaseServerId string         `json:"dbServerId"`
```

Comment:
I assume this is dbserver id. the "DbServer" field here has an Id. In addition to that, do we still need to store that id as a separate field?